CustomApps/new-releases/Card.js

```diff
@@ -19,6 +19,11 @@ class Card extends react.Component {
 		event.stopPropagation();
 	}
 
+	closeButtonClicked(event) {
+		removeCards(this.props.uri);
+		event.stopPropagation();
```

Comment:
why wouldnt it be? i need to pass the uri to the index and stop the event afterwards, this also means if it errors it just opens the cards page instead of crashing spotify.